server/game/cards/06-CotE/AspirantChallenger.js

```diff
@@ -0,0 +1,33 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+
+class AspirantChallenger extends DrawCard {
+    setupCardAbilities(ability) {
+        this.composure({
+            effect: ability.effects.modifyMilitarySkill(2)
+        });
+        this.action({
+            title: 'Initiate a Military duel',
+            condition: context => context.source.isParticipating(),
+            target: {
+                cardtype: CardTypes.Character,
```

Comment:
cardtype should be cardType - ts won't let you make this mistake any longer :P

Reply:
haha.  Glad to hear it.  Fixed.